bitbots_blackboard/src/bitbots_blackboard/capsules/world_model_capsule.py

```diff
@@ -482,7 +495,10 @@ def localization_pose_current(self) -> bool:
         """
         Returns whether the last localization pose was received in the last pose_lost_time-setting seconds.
```

Comment:
You are not using the `pose_lost_time` anymore?

Reply:
yes, its removed